### DISCUSSION TITLE
r/live-on-the-rv-during-the-spring-festival

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#14] [Announcements]
# r/live-on-the-rv-during-the-spring-festival

这个春节我们住在房车上

https://blog.ops-coffee.cn/r/live-on-the-rv-during-the-spring-festival

**Comment by yimkong** [comment]
挺有意思的，也是一种生活体验

**Comment by NangeGao** [comment]
真棒，看得心痒。我老家小村比你家更北更冷，但我在深圳，过年那几天回家体验真的一言难尽

**Reply by ops-coffee** [reply]
自从买了房车之后，每次回家都是住在车上的，已经习惯了，挺好
